borgmatic/tests/unit/borg/test_check.py

```diff
@@ -89,7 +89,7 @@ def test_make_check_flags_with_default_checks_and_last_returns_last_flag():
 )
 def test_check_archives_calls_borg_with_parameters(checks):
     check_last = flexmock()
-    consistency_config = flexmock().should_receive('get').and_return(check_last).mock
+    consistency_config = {'check_last': check_last}
```

Comment:
I removed the flexmock usage here. It felt as though it was not adding value. The method being tested should transform config into an output, it doesn't make sense for the test to be concerned about what keys the _black box_ method is using from the config object passed

Reply:
Yup, makes sense. Thanks!